src/main/java/net/snowflake/ingest/SimpleIngestManager.java

```diff
@@ -625,6 +621,46 @@ public HistoryRangeResponse getHistoryRange(
     return ServiceResponseHandler.unmarshallHistoryRangeResponse(response);
   }
 
+  /**
+   * Register a snowpipe client and returns the client sequencer
+   *
+   * @param requestId a UUID we use to label the request, if null, one is generated for the user
+   * @return
+   * @throws URISyntaxException
+   * @throws IOException
+   * @throws IngestResponseException
+   */
+  public ConfigureClientResponse configureClient(UUID requestId)
+      throws URISyntaxException, IOException, IngestResponseException {
+    if (requestId == null) {
```

Comment:
Lets check for emptiness too.. 
```suggestion
    if (requestId == null || requestId.toString().isEmpty()) {
```